tests/pos/i3471.scala

```diff
@@ -0,0 +1 @@
+sealed object Fun
```

Comment:
Shouldn't we also print a warning saying that the modifier is useless here?

Reply:
We don't do it for final either. If we solve it we should solve it for both at the same time. But not that scalac does not warn for `final object` either.